### DISCUSSION TITLE
How to know if the i set the server correct

### DISCUSSION THREAD

**Discussion by raphalimam** [Impostor/Impostor#243] [Q&A] [answered]
Hi i'am trying to create a serve to se if it works so i can get my friends to play some custom games, but the game dont show anything that make me certain that is working as i try to create a game.

![Captura de Tela (12)](https://user-images.githubusercontent.com/15850222/102540691-2843d580-408e-11eb-85a0-4a7bb733f8a4.png)
![Captura de Tela (13)](https://user-images.githubusercontent.com/15850222/102540694-29750280-408e-11eb-8994-473809e0cdcd.png)

**Comment by AeonLucid** [comment, marked as answer]
In the bottom left it should show "Impostor" where it shows "North America" now.
You need to use the patcher, for which you can find instructions [here](https://github.com/Impostor/Impostor#windows).

**Reply by raphalimam** [reply]
i thought that the server creator didint need to use the client, thanks, but i keep getting disconnected as i host the room, i think maybe is the version, i will test latter, but thanks

**Comment by klugemonkey** [comment]
This is confusing as well for me.  Is this really a server, meaning doesn't need anyone client to host or is it just a UDP proxy to bypass official servers and setup a private lan party.  For me, a server actually means everyone can connect to the server and no one player has to be hosting the game.
